docs/docusaurus.config.js

```diff
@@ -30,6 +30,7 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
+          breadcrumbs: true,
```

Comment:
Perhaps it wasn't so helpful before as there was just two layers, but now there is more.